lib/drizzlepac/catalogs.py

```diff
@@ -740,13 +741,15 @@ def plotXYCatalog(self, **kwargs):
 
         """
         try:
-            from matplotlib import pyplot as pl
+            from matplotlib import pyplot as plt
         except:
-            pl = None
+            plt = None
 
-        if pl is not None:
-            pl.clf()
-            pl.plot(self.xypos[0],self.xypos[1],**kwargs)
+        if plt is not None:
+            plt.clf()
+            plt.plot(self.xypos[0],self.xypos[1],**kwargs)
+
+            plt.figure()  # create new figure for subsequent plots (by user?)
```

Comment:
Shouldn't figure() be called before plot()? CC: @sosey

Reply:
arg, I can't believe the comments got deleted :(   attempting to recreate....